src/Umbraco.Web.UI.Client/src/common/services/umbdataformatter.service.js

```diff
@@ -425,7 +426,15 @@
                         var variant = displayModel.variants[j];
 
                         _.each(invariantProperties, function (invProp) {
-                            variant.tabs[invProp.tabIndex].properties[invProp.propIndex] = invProp.property;
+                            var tab = variant.tabs[invProp.tabIndex];
+                            var prop = tab.properties[invProp.propIndex];
+
+                            if (prop.segment) {
+                                // Do not touch segmented properties
```

Comment:
Not sure I understand this check. If we have a non-culture + segmented property on a culture document then the segmented property would need to be the same reference across all cultures?